examples/generate.py

```diff
@@ -0,0 +1,47 @@
+'''Generate example page with overview of all icons.'''
```

Comment:
Better to name this file to something like `update_icons.py`.

Reply:
done